src/components/Handlers/Handlers.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withModules } from '../Modules';
+import { stripesShape } from '../../Stripes';
+
+class Handlers extends React.Component {
+  static propTypes = {
+    stripes: stripesShape.isRequired,
+    event: PropTypes.number,
+    data: PropTypes.object,
+    modules: PropTypes.shape({
+      handler: PropTypes.array,
+    }),
+  };
+
+  constructor(props) {
+    super(props);
+
+    const { event, stripes, modules, data } = props;
+
+    this.components = modules.handler.reduce((acc, m) => {
```

Comment:
This is delightfully simple and elegant. Seeing the code makes me more confident that we got the abstraction right.